specifyweb/frontend/js_src/lib/components/WbPlanView/Mapper.tsx

```diff
@@ -480,7 +493,7 @@ export function Mapper(props: {
       <Ul
         aria-label={wbPlanText.mappings()}
         className={`
-          grid flex-1 auto-rows-max grid-cols-[theme(spacing.8)_max-content_auto]
+          grid flex-1 auto-rows-max grid-cols-[theme(spacing.24)_max-content_auto]
```

Comment:
The change from 8 to 24 could be a bug. Could you try making the side not be hardcoded but automatic? And if not, then this size should be different depending on whether one delete button or two buttons are displayed